examples/Testofparameters.jl

```diff
@@ -0,0 +1,104 @@
+using MPSGE
+# m = Model()
+
+# @parameter(m, endow, 1.0)
+# @parameter(m, diff, 0.0)
+
+# @sector(m, X)
+# @sector(m, Y)
+# @sector(m, U)
+
+# @commodity(m, PX)
+# @commodity(m, PY)
+# @commodity(m, PU)
+# @commodity(m, PL)
+# @commodity(m, PK)
+
+# @consumer(m, RA, benchmark = 150.)
```

Comment:
Why is there so much code that is commented out? Should this file even committed at all, I think it would be much cleaner if a PR like this one here just had changes to `src` and the `test` directory. This looks more like a scratch file?